README.md

```diff
@@ -61,28 +81,38 @@ start, by running:
 ```
 docker run -p 9944:9944 local/mashnet-node [node command]
 ```
+For execution see the section about [Commands](#commands).
 
 #### Build code without docker
-You need to have rust and cargo installed and configured properly.
+You need to have [rust and cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) installed. Clone this repo and navigate into it.
 
 You can build it by executing these commands:
 ```
 ./init.sh
 ./build.sh
 ```
-for building in dev mode
+#### Building in dev mode
 ```
 cargo build
 ```
-for building in release mode
+start, by running:
+```
+./target/debug/node [node command]
+```
+
+#### Building in performant release mode
 ```
 cargo build --release
 ```
-For execution see the section about commands.
+start, by running:
+```
+./target/release/node [node command]
+```
 
+For execution see the section about commands.
 ### Commands
 To start the node you have following options:
-- start-node.sh helper script
+- st<span>art</span>-node.sh helper script
```

Comment:
Small typo here?

Reply:
That's so start-node.sh doesn't show up as a link

Reply:
Does it?

Displayed as follows on chrome, macOS:

![Screenshot 2019-11-11 at 14 18 32](https://user-images.githubusercontent.com/9762897/68590164-2c1a2700-048e-11ea-90a1-5241d8b9665b.png)

Reply:
uhh yeah github displays t correctly, vanilla markdown would see it as a link